src/vulkan/CMakeLists.txt

```diff
@@ -32,6 +32,14 @@ set(VULKAN_ENGINE_SOURCES
     vertex_buffer.cc
 )
 
+if (${AMBER_LOG_VULKAN})
```

Comment:
I think we want to add an option(AMBER_LOG_VULKAN ...) into the top level CMakeLists.txt file (similar to AMBER_SKIP_SAMPLES etc)